docker-compose.yml

```
@@ -0,0 +1,8 @@
+services:
+  database:
+    container_name: "postgres_dev"
+    image: "postgres:17.2-alpine3.21"
+    env_file:
+      - "./.env.development"
+    ports:
+      - "5432:5432" 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)